### ISSUE TITLE
Adding other methods for A.I. generation.

### ISSUE THREAD

**Issue by learnitall** [learnitall/TicTacTio#8]
In order to explore the world of machine learning (and to create a somewhat good A.I. for the game itself), I would like to add two more A.I.s: one done with gradient descent and one done by using an algorithm/workflow. It would then be awesome to create a script that would compare their efficiency and skill.

**Comment by learnitall** [comment]
Not sure if this really fits here, but I'm going to add both multiprocessing and openGL support to A.I. generation. Right now trying to generate an A.I. with a large population of neural nets is pretty slow, so this could help speed things up a bit.

**Comment by learnitall** [comment]
Last night at around 1 A.M. I finally finished adding multiprocessing (tried three different methods and finally got the third one to work properly). It's weird though that each cpu is only working at around 60-70%, so I'm going to look into what could be causing the inefficiencies.
